docs/Makefile

```diff
@@ -0,0 +1,23 @@
+DOCREPO := github.com/keptn-sandbox/lifecycle-toolkit-docs
+TMPDIR := $(CURDIR)/tmp
+VOLUMES := -v $(TMPDIR)/lifecycle-toolkit-docs:/src -v $(TMPDIR)/lifecycle-toolkit-tmp/public:/target -v $(CURDIR)/en:/src/content/en
+IMAGE := klakegg/hugo:0.105.0-ext
```

Comment:
pls add a renovate comment here. i guess this should be updated with docker directly instead of the github repo